### PR TITLE
some updates to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,10 +74,14 @@ and
...
+This is an incremental process.
+If you're adding new code, adhering to the formatting config is appreciated.
+Formatting breakage in already formatted files will be caught by the CI.
+
+Additional guidelines:
 
 * Don't have end-of-line whitespace
 * Use spaces instead of tabs
-* Stick to the style of the file you're editing
 * Functions and classes must have a [docblock](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html)
```
Comment (omoerbeek) [inline CONTRIBUTING.md:86]: Is docblock still relevant?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,10 +74,14 @@ and
 * Do not put whitespace fixes/cleanup and functionality changes in the same commit
 
 # Coding Guidelines
-At the moment there is no established coding guideline, but here are some
-general guidelines:
+We have `clang-format` in place, but not for all files yet.
+This is an incremental process.
+If you're adding new code, adhering to the formatting config is appreciated.
+Formatting breakage in already formatted files will be caught by the CI.
```
Comment (omoerbeek) [inline CONTRIBUTING.md:80]:
```suggestion
Formatting breakage in already formatted files will be caught by the CI.
To format all files that are supposed to be formatted, run `make format-code` in the root of the tree.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (Habbie) [reply CONTRIBUTING.md:86]: We all suck at doing it. Perhaps it's unfair to ask this from contributors, yes.

Comment (Habbie) [reply CONTRIBUTING.md:87]: also the link is 404

Comment (rgacogne) [reply CONTRIBUTING.md]: Oh, crap, I was supposed to do that? :-D

Comment (Habbie) [reply CONTRIBUTING.md]: pushed new text

**Review by omoerbeek — APPROVED** [review]

**Pull request merged**